tests/codes/test_latex_formula.py

```diff
@@ -32,3 +32,15 @@ def test_str(self, fixture_latex_formula: LatexFormula) -> None:
         # Expected result
         expected_result = "E = mc^2 = 5*10^2 = 500"
         assert str(fixture_latex_formula) == expected_result
+
+
+def test_latex_fraction() -> None:
+    """Test the latex_fraction function."""
```

Comment:
```suggestion
    """Test the latex_fraction function."""
```
This name (latex_fraction) is much better, but it it is different from the function you are testing.